xbmc/music/windows/GUIWindowVisualisation.cpp

```diff
@@ -240,3 +241,29 @@ void CGUIWindowVisualisation::FrameMove()
   }
   CGUIWindow::FrameMove();
 }
+
+void CGUIWindowVisualisation::ToggleOSD()
+{
+  CGUIDialogMusicOSD *pOSD = static_cast<CGUIDialogMusicOSD *>(g_windowManager.GetWindow(WINDOW_DIALOG_MUSIC_OSD));
+  if (pOSD)
+  {
+    if (pOSD->IsDialogRunning())
+    {
+      pOSD->Close();
```

Comment:
Do not understand this, sorry. I can only say that it works and that this method gets called. ???